app/views/callbacks/steps/_personal_details.html.erb

```diff
@@ -9,3 +9,8 @@
 <%= f.govuk_text_field :first_name, autocomplete: "given-name" %>
 <%= f.govuk_text_field :last_name, autocomplete: "family-name" %>
 <%= f.govuk_email_field :email, autocomplete: "email" %>
+<%= f.hidden_field :accepted_policy_id, value: f.object.latest_privacy_policy.id %>
+
+<p>Your details are protected under the terms of our <%= link_to("privacy notice", privacy_policy_path(id: f.object.latest_privacy_policy.id), { target: :blank }) %>.</p>
+
+<p>Our privacy notice explains how we use your personal data. It is important you have read this notice before signing up for a callback.</p>
```

Comment:
```suggestion
<p>Our privacy notice explains how we use your personal data. It's important you read this before signing up.</p>
```
Nothing major to suggest - just a few potential tweaks? @gemmadallmandfe @philspray